EdgeMultiplay/Scripts/Util.cs

```diff
@@ -162,5 +164,56 @@ public static Quaternion ConvertFloatArrayToQuaternion(float[] floatArray, int s
                 throw new Exception("floatArray starting from the start index doesn't qualify to create a Vector3");
             }
         }
+
+        /// <summary>
+        /// Converts a Hashtable to Dictionary<string, string>
+        /// </summary>
+        /// <param name="htable">Hashtable</param>
+        /// <returns>Dictionary<string, string></returns>
+        public static Dictionary<string, string> HashtableToDictionary(Hashtable htable)
+        {
+            Dictionary<string, string> dict = new Dictionary<string, string>();
+            Debug.Log("HashtableToDictionary: " + htable + ", Count: " + htable.Count);
+            if (htable == null || htable.Count == 0)
+            {
+                return dict;
+            }
```

Comment:
hmm this should return null if htable is null. The code you had before was good, you just had to remove of the check for htable.Count==0